src/main/java/softdesign/map/CartesianCoordinate.java

```diff
@@ -0,0 +1,23 @@
+package main.java.softdesign.map;
+
+import javax.vecmath.Point3d;
+import main.java.softdesign.Environment;
+
+public class CartesianCoordinate {
```

Comment:
Insert an empty line before declaring the variables to stay consistent with the code base.